docs/developers/celestia-app-vesting.md

```diff
@@ -0,0 +1,404 @@
+---
+sidebar_label: How to create a vesting account with Celestia App
+description: Learn how to generate a vesting account using Celestia App.
+---
+
+# How to create a vesting account with Celestia App
+<!-- markdownlint-disable MD013 -->
+
+In this guide, we will learn how to create a vesting account using
+celestia-appd for both a local devnet and on Mocha testnet.
+
+## Local devnet
+
+First, download and [install Celestia App](../../nodes/celestia-app),
+selecting the
+[network and corresponding version](../../nodes/participate)
+that you would like to use.
+
+### Setting up the local devnet
+
+#### Run the devnet
+
+Next, change into the `$HOME/celestia-app` directory and
+run the single-node-devnet script.
+
+```bash
+cd $HOME/celestia-app
+./scripts/build-run-single-node.sh
+```
+
+#### Save the home directory path
+
+At the top of the output, you will see a path to the "Home directory",
+find yours from the output (**it will be unique every time**):
+
+```bash
+./scripts/build-run-single-node.sh
+Home directory: /var/folders/_8/ljj6hspn0kn09qf9fy8kdyh40000gn/T/celestia_app_XXXXXXXXXXXXX.XV92a3qx
+--> Updating go.mod
+```
+
+And set the location as the `NODE_STORE` variable. We will use this
+for the remainder of the devnet section.
+
+```bash
+export NODE_STORE=/var/folders/_8/ljj6hspn0kn09qf9fy8kdyh40000gn/T/celestia_app_XXXXXXXXXXXXX.XV92a3qx
```

Comment:
[nit] is it possible to use a different variable name? `NODE_STORE` makes me think this is related to celestia-node but this variable is for the celestia-app home.

When I do this locally, I usually name it `CELESTIA_APP_HOME` but up to you.

Reply:
definitely possible, i'll use `CELESTIA_APP_HOME` thanks!

Reply:
resolved in 8adf1efdb71f53871914f5fa87054a0dc4c6e216